protos/sources/vega/commands/v1/commands.proto

```diff
@@ -304,6 +304,12 @@ message Transfer {
   string amount = 5;
   // Reference to be attached to the transfer.
   string reference = 6;
+  // Optional derived subaccount key from which assets are transferred. 
+  // The submitter of the transaction must be the owner of this derived subaccount key.
+  // If provided, the 'from_account_type' must be REWARDS_VESTED, and the asset in this account 
+  // must match the asset specified in the transfer.
+  optional string from_sub_account = 7;
```

Comment:
```suggestion
  optional string from = 7;
```
removing sub-account here we're tidying up the use of sub-account, which is used to mean several different things